engine/Assets/UI/Screen.uxml

```diff
@@ -1,13 +1,14 @@
 <ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements">
     <ui:VisualElement name="screen" class="main-screen" style="flex-grow: 1;">
-        <Style src="/Assets/UI/Screen.uss" />
+        <Style src="Screen.uss" />
         <ui:VisualElement name="catch-all-mouse-down" style="width: 100%; height: 100%; position: absolute; left: 0; right: 0; top: 0; bottom: 0;" />
-		<ui:VisualElement name="toast-notification-container" style="position: absolute; left: 0; bottom: 0; width: 400px; height: 0; max-height: 400px; overflow: hidden; margin-left: 30px; margin-right: 30px; margin-top: 30px; margin-bottom: 30px;">
+        <ui:VisualElement name="dialog-free-roam" style="position: absolute; width: 100%; height: 100%;" />
```

Comment:
You may want to move this to the bottom of the "screen" element so that it will appear on top of everything else.